grid/booking/list-activities/maps/list-activities.ts

```diff
@@ -24,6 +24,31 @@ export function listActivitiesTest(provider: string) {
         expect(() => result.unwrap()).not.toThrow();
         expect(result).toMatchSnapshot();
       });
+
+      it('should paginate', async () => {
+        const result = await superface.run({
```

Comment:
nitpick: Rename `result` to `firstResult`

Reply:
Sure